ext/standard/config.w32

```diff
@@ -1,6 +1,18 @@
 // vim:ft=javascript
 // $Id$
 
+ARG_WITH("argon2", "Argon2 support", "no");
+
+if (PHP_ARGON2 != "no") {
+	if (CHECK_LIB("Argon2Ref.lib", "argon2", PHP_ARGON2)
+	&& CHECK_HEADER_ADD_INCLUDE("argon2.h", "CFLAGS_ARGON2")) {
+		AC_DEFINE('HAVE_ARGON2LIB', 1);
+		EXTENSION("argon2", null, false);
```

Comment:
I was getting linker errors without the call to `EXTENSION`. Is there another macro I can use to get `Argon2Ref.,lib` to be added to the linker? I wasn't seeing any other function for handling that.